### PR TITLE
Add spec_cleaner/__main__.py to allow running spec-cleaner without installing it

### PR DESCRIPTION
**Pull request by peti**
Running `python3 -m spec_cleaner` from the top-level directory of the Git repository is going to work fine now; no more wrapper script is necessary.

### CODE REVIEW

File: spec_cleaner/__main__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+
+import os
+import sys
+
+# If we are running from a wheel, add the wheel to sys.path
```
Comment (aplanas) [inline spec_cleaner/__main__.py]: I guess that is not adding the wheel but the wheel / package directory?

**Comment by matejcik** [comment]
it would be nice to `s/pip/spec_cleaner/` in the comments ;) otherwise LGTM

File: spec_cleaner/__main__.py (hunk 1 above)
Comment (matejcik) [inline spec_cleaner/__main__.py]: no, the comments are correct, this will add the `whl` file

**Comment by peti** [comment]
I've cleaned up the comments and force-pushed a new version.

**Pull request merged**